dask_cuda/cli/dask_cuda_worker.py

```diff
@@ -277,6 +277,13 @@
     help="""Use a different class than Distributed's default (``distributed.Worker``)
     to spawn ``distributed.Nanny``.""",
 )
+@click.option(
+    "--pre-import",
+    default=None,
+    help="""Pre-import libraries as a Worker plugin to prevent long import times
+    bleeding through later Dask operations. Should be a list of comma-separated names,
+    such as "cudf,rmm".""",
```

Comment:
Apologies for the drive-by comment, but wouldn't a `List[str]` be more appropriate here?

Reply:
There is [`nargs` support]( https://click.palletsprojects.com/en/8.0.x/options/#multi-value-options ) in Click. Is that what you are referring to?

Reply:
Ah hmm that wouldn't work either as it's not a fixed number of args. I'm just going to close this suggestion as I don't think there's a better way to do it, honestly.

Reply:
Is it? I think if [`nargs=-1`]( https://click.palletsprojects.com/en/8.0.x/arguments/#variadic-arguments ), we'd get the intended behavior

Reply:
`nargs=-1` seems to only apply to variadic arguments though (i.e., "unnamed" arguments), but [not support for options](https://github.com/pallets/click/blob/main/src/click/core.py#L2492) as is the case for `--pre-import`.

Reply:
Makes sense. Then let's leave as-is. There doesn't seem to be a good alternative